x/observer/keeper/migrator.go

```diff
@@ -61,3 +62,8 @@ func (m Migrator) Migrate8to9(ctx sdk.Context) error {
 func (m Migrator) Migrate9to10(ctx sdk.Context) error {
 	return v10.MigrateStore(ctx, m.observerKeeper)
 }
+
+// Migrate10to11 migrates the store from consensus version 10 to 11
+func (m Migrator) Migrate10to11(ctx sdk.Context) error {
```

Comment:
Will this go to the release to be cut next Monday? If it is, you could just reuse the existing `Migrate9to10`, otherwise just ignore this comment.

Reply:
I'd prefer to keep the migrations separate as the `Migrate9to10` has already run on developnet. These migrations are very light so I don't think we need to worry about it.

If we were doing a pass over all CCTX or something heavy like that I think it would be worth considering.

Reply:
okay.

Reply:
I agree with the original comment; It would be better to reuse the existing migration if  possible. 
Assuming developnet can be reset easily ?

If not, it would be great to atleast manually check that both the migrations are running and all the values are correctly set

Reply:
I can reset it but it takes some time